packages/app/src/update/actions/start-update-os/start-update-os.action.ts

```diff
@@ -55,6 +55,7 @@ export const startUpdateOs = createAsyncThunk<
     let state = getState()
     const batteryLevel = state.device.data?.batteryLevel ?? 0
     const deviceType = state.device.deviceType
+    const externalUsageDevice = state.device.externalUsageDevice
```

Comment:
🟢 
```suggestion
    const externalUsageDevice = state.device.externalUsageDevice ?? false
```

Reply:
resolved in 94eb632df33bbd05fa6bba41bcb9482d6d1cfcd5